autogpts/autogpt/autogpt/url_utils/validators.py

```diff
@@ -38,6 +41,11 @@ def wrapper(url: str, *args, **kwargs) -> Any:
         # Check URL length
         if len(url) > 2000:
             raise ValueError("URL is too long")
+        if web_policy:
+            if url not in url_list:
+                raise ValueError("URL Not Whitelisted")
+        elif url in url_list:
+            raise ValueError("URL Blacklisted.")
```

Comment:
inconsistent formatting

Reply:
I have fixed the linting this update the PR with the above config changes